### PR TITLE
[ADF-5062] Content Node Selector - show inputs based on configuration

### PR DESCRIPTION
**Pull request by pionnegru**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
[ADF-5062](https://issues.alfresco.com/jira/browse/ADF-5062)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts [hunk 1]
```diff
@@ -54,6 +54,9 @@ export class ContentNodeSelectorPanelComponent implements OnInit, OnDestroy {
         hasMoreItems: false
     });
 
+    private _showDropdownSiteList = true;
```
Comment (eromano) [inline lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts]: I don't think we are using this naming convention of the underscore for private

Comment (mmuller88) [reply lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts]: If you expand a bit down there is one _chosenNode:

Comment (eromano) [reply lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts]: _chosenNode is different has a get and set in that case is needed

Comment (eromano) [reply lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts]: @pionnegru thanks

**Pull request merged**